src/panel/quick_settings/quick_settings_grid/quick_settings_grid_theme.c

```diff
@@ -5,6 +5,7 @@
 #include "../../../services/theme_service.h"
 #include "quick_settings_grid_button.h"
 
+/* old
```

Comment:
Hey @parmjotsinghrobot 

Just clean this up, and other then this I think the code looks good! Its working fine on my end :)